react/week1/listitems.js

```diff
@@ -0,0 +1,11 @@
+function ListItems(props) {
+    return (
+      <ul>
+        <li>
+         <h4> Task:{props.task}, Deadline:{props.deadline}</h4>
+        </li>
+      </ul>
```

Comment:
If you are using vs code, I suggest you use an extension called prettier. It will format your code automatically. 
You can set is as your default formatter in the vs code settings. You can also set it to format on save.